lib/unsupported_locales.js

```diff
@@ -0,0 +1,28 @@
+require('babel-core/register');
+
+var Parser = require("accept-language-parser");
+var bestLang = require('bestlang');
+var pages = Object.keys(require('../data/pages.js'));
+var supported_locales = require('../locales/index.js');
+var lastModified = (new Date()).toUTCString();
+
+module.exports = function(securityConfig) {
+  return pages.map(function(page) {
+    return {
+      method: 'GET',
+      path: '/{locales}' + page,
+      handler: function(request, reply) {
+        var query = '';
+        var header = Parser.parse(request.headers["accept-language"]);
+        var languages_array = header.map(function(l) {
+          return l.code + (l.region ? "-" + l.region : "");
+        });
+        if (request.url.search) {
+          query = request.url.search;
+        }
+        reply.redirect('/' + bestLang(languages_array, supported_locales, 'en-US') + page + query)
+        .vary("Accept-Language").header("Last-Modified", lastModified);
+      }
```

Comment:
Add a 1 day cache-control header here, so if we add a new language it doesn't take long to be enabled in the redirect